packages/web-components/src/components/va-checkbox/va-checkbox.scss

```diff
@@ -22,24 +22,12 @@
   margin: 0.75rem 0;
   max-width: 480px;
   cursor: pointer;
-
-  // Temporarily adding vars manually until Storybook is setup to make use of CSS-Library
-  --vads-button-color-background-primary-on-light: #005ea2;
-  --vads-input-background-color-on-light: #ffffff;
-  --vads-input-border-color-on-light: #565c65;
-  --vads-input-prefix-color-fill-on-light: #757575;
-  --vads-input-prefix-color-text-on-light: #757575;
-  --vads-input-suffix-color-text-on-light: #757575;
-  --vads-input-tile-background-active-on-light: rgba(0, 94, 162, 0.1);
-  --vads-input-tile-border-active-on-light: #005ea2;
-  --vads-input-tile-border-on-light: rgba(27, 27, 27, 0.03);
```

Comment:
All of these tokens are already being imported globally for the web-component files: https://github.com/department-of-veterans-affairs/component-library/blob/d04f16d914a8da3ad65dea40891a4a465822575e/packages/web-components/src/global/main.css#L1